src/lib.rs

```diff
@@ -497,10 +492,27 @@ impl Variant {
 /// * Deviations from standard.
 /// * No length limits are enforced for the data part.
 #[cfg(feature = "alloc")]
```

Comment:
In 1facf64fcd706c3c2a4f6c21349efcdcd308cd93:

You can remove the `cfg(alloc)` from this function now. You may also want to reformat the doccomment to have `Deviations from standard` be a heading, like you did on the other one.

Reply:
This is on `encode`, right? If so it returns a `String` (I checked all feature gating and it looks correct but I'm brain fried as f*** right now :)

Reply:
I fixed the docs.

Reply:
> This is on `encode`, right? If so it returns a `String` (I checked all feature gating and it looks correct.)

Derp, you're right.